README.md

```diff
@@ -1,5 +1,8 @@
 # Vector Remap Language (VRL)
 
+[![Crates.io](https://img.shields.io/crates/v/vrl?style=flat-square)](https://crates.io/crates/vrl)
+[![docs.rs](https://img.shields.io/docsrs/vrl?style=flat-square)](https://docs.rs/vrl/0.4.0/vrl/)
```

Comment:
There's another one to show the status of CI, would probably be nice to add.
See: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge
The badge: https://github.com/vectordotdev/vrl/actions/workflows/test.yml/badge.svg